.github/workflows/frontend.yml

```diff
@@ -0,0 +1,47 @@
+name: Frontend Workflow Check
+on: [pull_request, push]
+
+concurrency: preview-${{ github.ref }}
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install pnpm
+        run: npm install -g pnpm 
+        working-directory: ./frontend
+
+      - name: Install
+        run: pnpm install --frozen-lockfile
+        id: install
+        working-directory: ./frontend
+
+
+      - name: Run ESLint
+        run: pnpm run lint
+        working-directory: ./frontend
+
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

Comment:
might need to update the version of `actions/checkout` to `actions/checkout@v3`. confirm with other yaml files in `.github/workflows` directory